lib/mongodb.js

```diff
@@ -0,0 +1,5 @@
+const mongodb = require('mongodb');
+
+module.exports = {
+  mongodb: mongodb,
```

Comment:
Any particular reason for the extra level of nesting?

Reply:
So that we can actively set it in [the tests](https://github.com/share/sharedb-milestone-mongo/pull/12/files#diff-3fd1ec19aee1b793e7b5f7cd3b54d8585a5b23add521d57f6859faec96bc1952R49).